### PR TITLE
Use django-jsonfield-backport (and move build env to github actions)

### PR DESCRIPTION
**Pull request by lociii**
Well... this PR started with the discussion in #449
Because I was annoyed by the extremely slow build times on TravisCI, I decided to move the build env to GitHub actions.
Not a big deal I tought...

(Sidenote: Travis will more or less remove it's free tier, so the migration would have been necessary at some point in the future anyway)

During implementation I found out that the current build is broken for Django 3.0/3.1, see https://github.com/justquick/django-activity-stream/blob/04db1ed1fd8d355fd77e24741725d917f78e8928/tox.ini#L13-L14
This is now implicitly fixed by the move to GitHub actions.

The build env also defined django-jsonfield>=1.0.1, see https://github.com/justquick/django-activity-stream/blob/04db1ed1fd8d355fd77e24741725d917f78e8928/tox.ini#L17
which will break tests on postgres because it doesn't deserialize JSON anymore after it's 1.3.0 release on newer Django >= 3.1 which is also mentioned in #457

Long story short: There is no easy and proper way to make all of this work smoothly. Therefore I decided to include the next step and replace all currently used libs with django-jsonfield-backport.
This change addresses #472, #449 and #437

The library gets installed by the `jsonfield` extra of actstream and will be dynamically loaded when necessary.
Everything should be fully compatible with existing setups.

### CODE REVIEW

**Comment by lociii** [comment]
This will also make #473 obsolete. Sorry.

File: .github/workflows/workflow.yaml [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+        options: >-
+          --health-cmd="pg_isready"
+          --health-interval=10s
+          --health-timeout=5s
+          --health-retries=5
+
+      # mysql service
+      mysql:
+        # docker hub image
+        image: mysql:5.7
```
Comment (auvipy) [inline .github/workflows/workflow.yaml:57]: why not upgrading to mysql 8.0+

File: .github/workflows/workflow.yaml [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+          - sqlite
+          - mysql
+          - postgres
+
+    # additional service containers to run
+    services:
+      # postgres service
+      postgres:
+        # docker hub image
+        image: postgres:9.5-alpine
```
Comment (auvipy) [inline .github/workflows/workflow.yaml:39]: postgres:10+ubuntu would be great

File: tox.ini [hunk 3]
```diff
@@ -1,35 +0,0 @@
-[tox]
```
Comment (auvipy) [inline tox.ini:1]: should we delete tox right now?

**Review by auvipy — CHANGES_REQUESTED** [review]

File: .github/workflows/workflow.yaml (hunk 1 above)
Comment (lociii) [reply .github/workflows/workflow.yaml:57]: These are the same requirements as in the tox build. Will happily upgrade to newer releases

File: .github/workflows/workflow.yaml (hunk 2 above)
Comment (lociii) [reply .github/workflows/workflow.yaml:39]: These are the same requirements as in the tox build. Will happily upgrade to newer releases

File: tox.ini (hunk 3 above)
Comment (lociii) [reply tox.ini:1]: I don't see a reason to continue using tox as GitHub actions can cover everything required.
Less tools, less trouble.

File: .github/workflows/workflow.yaml (hunk 2 above)
Comment (lociii) [reply .github/workflows/workflow.yaml:39]: Any specific reasons for ubuntu as alpine images are smaller, lightweight and faster.

File: tox.ini (hunk 3 above)
Comment (lociii) [reply tox.ini:1]: You can even run GitHub actions locally in a proper way with https://github.com/nektos/act
This will also use the proper Python versions whereas tox locally uses the installed 3.x release for all 3.x tests as it's based on virtualenv. So no proper 3.6, 3.7, 3.8, etc tests locally with tox.

**Comment by lociii** [comment]
Updated postgres to 13 and mysql to 8

**Review by auvipy — APPROVED** [review]
